### ISSUE TITLE
Colorful rectangles around code instead of underlines in last update (1.5.1)

### ISSUE THREAD

**Issue by widersky** [subtheme-pro/atom-one#9]
**Describe the bug**
I think screenshot below describes this issue best:

![Zrzut ekranu 2022-05-19 o 18 23 38](https://user-images.githubusercontent.com/45122321/169350456-e3afbf35-f40f-47ee-8440-0fa789499f60.png)

**To Reproduce**
1. Update IDE to latest version (WebStorm 2022.1.1 here)
2. Install Atom One Theme
3. Activate
4. Open any file with warnings or errors present

**Screenshots**
![Zrzut ekranu 2022-05-19 o 18 23 38](https://user-images.githubusercontent.com/45122321/169350456-e3afbf35-f40f-47ee-8440-0fa789499f60.png)
![Zrzut ekranu 2022-05-19 o 18 29 03](https://user-images.githubusercontent.com/45122321/169351153-31595a02-99ef-4ade-a59f-911e16dff3c2.png)

**Application:**
* Intellij Idea 2022.1.1
* WebStorm 2022.1.1

**Comment by devleesch001** [comment]
I think it's not a BUG but a choice.
I prefer the older

**Comment by widersky** [comment]
@devleesch001 in light theme version this issue is not appearing so I think it's bug (I hope so!). I strongly prefer the older appearance too - current looks crappy

**Comment by BarryTong98** [comment]
Do we have any way to degrade the Atom one Theme from 1.5.1 to 1.4?

**Comment by widersky** [comment]
@BarryTong98 I was searching for this but without luck :/

**Comment by BarryTong98** [comment]
> @BarryTong98 I was searching for this but without luck :/

I just found it and degrade from 1.5.1 to 1.4. You have Plugin Versions in the JetBrains -> [https://plugins.jetbrains.com/plugin/14799-atom-one-theme/versions](url)

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/67917837/170295078-b34a9844-578d-4307-95c3-a76443f8ddfd.png">
Download the 1.4 version and choose install the plugins from disk
<img width="720" alt="image" src="https://user-images.githubusercontent.com/67917837/170295359-b878f346-3e62-453b-bdda-76780fd7ce41.png">

**Comment by mscott314** [comment]
You can customize these too.
![Screenshot_20220711_163340](https://user-images.githubusercontent.com/46970185/178353358-09115c4d-3b43-468b-b17b-8dd8ccb1cddf.png)

**Comment by devleesch001** [comment]
yes I know but I don't want to change it on all my IDEs, on all my machines and with each reinstallation
